src/gui/MainWindow.cpp

```diff
@@ -1019,8 +1019,11 @@ void MainWindow::OnConsoleLanguage(wxCommandEvent& event)
 	default:
 		cemu_assert_debug(false);
 	}
-	m_game_list->DeleteCachedStrings();
-	m_game_list->ReloadGameEntries(false);
+	if(!IsGameLaunched())
```

Comment:
```suggestion
	if (m_game_list)
```
Probably better to just check if the list exists?